lib/capybara/webkit/node.rb

```diff
@@ -47,6 +47,19 @@ def set(value)
       invoke "set", *[value].flatten
     end
 
+    def send_keys(*keys)
+      invoke("sendKeys", [keys].flatten.map { |key|
+        case key
+        when :space
+          " "
+        when :enter
+          "\r"
+        else
+          key.to_s
```

Comment:
If somebody uses a different special character, like `:left` or `[:control, "a"]`, this will type those out in the browser, which I worry is confusing. I'd rather fail loudly when the user tries to do something we don't support yet. Here's my suggestion:

* Change this `else` to `case String`.
* Add an `else` that raises `Capybara::NotSupportedByDriverError`.
* Remove the `flatten` so that arrays will also cause an error, as we don't really support them.

What do you think?